lib-ghe.sh

```diff
@@ -4,7 +4,7 @@
 
 # Default constants
 LOG='log'
-REQUIRED_GHE_MAJOR_VERSION='2.10'
+REQUIRED_GHE_MAJOR_VERSION='2.11'
```

Comment:
it's not that easy. most scripts work but I have to adjust the github-audit scripts as Github changed their location.

/cc @bbatha